src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -366,14 +883,19 @@ tests:
           and the scene identifier to field set to 0xFE."
       PICS: S.S.C40.Rsp
       verification: |
-          ./chip-tool scenesmanagement copy-scene 0 0x0000 0x01 0x4E20 sceneFE [] 1 1
-
-          CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0005 Command 0x0000_0040
-          CHIP:TOO:   CopySceneResponse: {
-          CHIP:TOO:     status: 137
-          CHIP:TOO:     groupID: 0
-          CHIP:TOO:     sceneID: 0xFE
-          CHIP:TOO:    }
+          ./chip-tool scenesmanagement store-scene 0x0000 0x01 3 1 --commissioner-name gamma
```

Comment:
Step 8 says "TH sends a CopyScene command to DUT with the" Why are you sending the `store-scene` command